README.md

```diff
@@ -161,7 +161,27 @@ Composer works great in combination with [Swarmer][swarmer] — another tool we'
 
 ### How to build
 
-Dependencies: you only need `docker` and `bash` installed on your machine.
+#### To build composer
+
+environmental variable ANDROID_HOME must be set.
+
+```console
+./gradlew build
+```
+
+#### To build html-report
+
+you need `npm` and `nodejs` packages installed.
+
+```console
+cd html-report
+npm install
+npm build
+```
+
+#### All-in-one script (used in Travis build)
```

Comment:
Can we put this one at the top of "How to build"? I don't want people install npm and stuff to build Composer